src/webhook/webhook.controller.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { WebhookController } from './webhook.controller';
+import { WebhookService } from './webhook.service';
+import { WebhookRepository } from './webhook.repository';
+import { HttpModule } from '@nestjs/axios';
+import { PrismaService } from '../prisma/prisma.service';
+
+describe('WebhookController', () => {
+  let controller: WebhookController;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [HttpModule],
+      controllers: [WebhookController],
+      providers: [WebhookService, WebhookRepository, PrismaService],
+    }).compile();
+
+    controller = module.get<WebhookController>(WebhookController);
+  });
+
+  it('should be defined', () => {
+    expect(controller).toBeDefined();
+  });
```

Comment:
would be good to add some tests

Reply:
Mentioned this in comment above PR post. (tests are in next PR 😉)